Examples/src/components/Examples/FeaturedApps/ScientificCharts/TenorCurves3D/angular.ts

```diff
@@ -0,0 +1,67 @@
+import { draw3DChart, drawLineChart1, drawLineChart2, drawHeatmapLegend } from "./drawExaple";
+import { Component } from "@angular/core";
+import { SciChartSurface, SciChart3DSurface } from "scichart";
+
+// @ts-ignore
+// import { drawExample } from "./drawExample";
+
+SciChartSurface.loadWasmFromCDN();
+SciChart3DSurface.loadWasmFromCDN();
```

Comment:
Hm, the usage of these 2  `loadWasmFromCDN` lines looks a bit inconsistent across different examples.
I think it's better to remove them from here (and from every angular.ts file)
Then we can use them from within a consuming component if needed.